tests/test_utils.py

```diff
@@ -26,6 +26,7 @@ def test_getting_metadata_from_content(self):
         post = dummy()
         post.source_path = 'file_with_metadata'
         post.metadata_path = 'file_with_metadata.meta'
+        post.default_lang = 'en'
```

Comment:
Would be nicer to make a special `post_dummy` object with `default_lang = 'en'` already.